packages/core/src/router/metadata.ts

```diff
@@ -167,3 +169,103 @@ export function operation(verb: string, path: string, spec: OperationObject) {
     );
   };
 }
+
+/**
+ * Describe an input parameter of a Controller method.
+ *
+ * @param paramSpec Parameter specification.
+ */
+export function param(paramSpec: ParameterObject) {
```

Comment:
`@param` is covered by a unit-test - see https://github.com/strongloop/loopback-next/blob/999424c97d28adb580d26000ca96594890248ee5/packages/core/test/unit/router/metadata/param.test.ts#L12-L33.